docs/messaging/kafka-integration.md

```diff
@@ -312,9 +324,7 @@ The .NET Aspire Apache Kafka integration uses the following log categories:
 
 #### Tracing
 
-The .NET Aspire Apache Kafka integration emits the following tracing activities using OpenTelemetry:
-
-- `Aspire.Confluent.Kafka`
+The .NET Aspire Apache Kafka integration dos not emit distributed traces.
```

Comment:
Good catch, metrics are included though already - right?

https://github.com/dotnet/aspire/blob/4e7e0de000bb83f824cca2773892a22b5da89245/src/Components/Aspire.Confluent.Kafka/AspireKafkaProducerExtensions.cs#L130

If so, I'll need to add that.